sky/provision/instance_setup.py

```diff
@@ -255,6 +268,8 @@ def start_ray_on_head_node(cluster_name: str, custom_resource: Optional[str],
     # Reference: https://github.com/skypilot-org/skypilot/issues/2441
     cmd = ('ray stop; unset AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY; '
            'RAY_SCHEDULER_EVENTS=0 RAY_DEDUP_LOGS=0 '
+           'RAY_LOG_TO_DRIVER_EVENT_LEVEL="FATAL" '
+           'RAY_BACKEND_LOG_LEVEL="FATAL" '
```

Comment:
We need comments somewhere in terms of why default would produce some undesirable outputs. (Which is odd; is this a latest Ray bug? Normally outputs should be useful at default logging level.)

Reply:
After we redirect the stderr, this log level seems not needed. Just removed : )